front/src/BottomBar/index.js

```diff
@@ -14,79 +14,119 @@ import pageTypes from "./pageTypes";
  * and the implementation of the functions passed to BottomBar.
  * Author: T.T.P. Franken
  * Author: B. Afonins
+ * Author: G.M. van der Sanden
  *
  * Props:
  * - onSend: function to send a command
  * - changeActivePage: function to change the active page
  * - activePage: a string indicating the current active page
+ * - isConnected: a boolean indicating whether the client is connected to the TRC
  */
 
 const BottomBar = props => {
-  const { changeActivePage, onSend, activePage } = props;
+  const { changeActivePage, onSend, activePage, isConnected } = props;
```

Comment:
you destruct `props` but dont use `isConnected` later in the code. change `props.isConnected` to just `isConnected`

Reply:
done